setup.cfg

```
@@ -1,3 +1,61 @@
+[metadata]
+name = pip
```

Maybe this would even work:
```suggestion
name = pip
version = attr: src.pip.__version__
```
(but probably harmful for backwards compat)

---

Won’t work on Python 2.7, so 🙅‍♂️🤷

---

I'd like to use `setuptools_scm` for version. Should we migrate to it?

---

I'm -1 on using `setuptools_scm`.